optimum/intel/openvino/configuration.py

```diff
@@ -40,7 +40,13 @@ class OVQuantizationMethod(str, Enum):
 
 
 _DEFAULT_4BIT_CONFIGS = {
-    "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "scale_estimation": True},
+    "databricks/dolly-v2-3b": {
```

Comment:
We will raise the bar on the HW requirements to do that kind of optimization as data-aware methods require more RAM and time to produce the result.

Reply:
What should we do then?

Reply:
This model is relatively small and may be it is ok to use data-aware configuration for it. Can you please share RAM and optimization time estimations for the config that you are pushing? If they are moderate we could probably keep your config.

Reply:
Okay. I'll re-calibrate this model along with the `4BIT_DEFAULT` configuration as the reference. Do I need to measure same numbers for other models in this PR?

Reply:
Here are graphs for the **dolly-v2-3b** model on the **main**:
![dolly-v2-3b_main](https://github.com/user-attachments/assets/c2ccdbe2-363c-4a6c-8c20-ce8d081f8c8e)
and **nm/fix_defaults** branches:
![dolly-v2-3b_branch](https://github.com/user-attachments/assets/1494a2ec-760a-4961-b560-6eaeacb49de2)
@AlexKoff88, @nikita-savelyevv

Reply:
It seems like 32GB is no longer enough to convert 3B model in this case which disrupts UX. @nikita-savelyevv, @alexsu52, can you please comment if we have plans to improve memory consumption in the nearest future?

Reply:
@AlexKoff88 There is some work planned in this direction but for now it is how it is. Also, the time increases about 10x, which is probably more critical.

Reply:
FYI, there are other options that exist. The closer one is to reducing the ratio to 0.7:

Config | Word PPL | Diff vs. FP
-- | -- | --
FP16, INT4_ASYM, GS   128, R 0.8 | 17,171 | 0,059685714
FP16, INT4_ASYM, GS 128, R 0.7 | 16,705 | 0,030933426
FP16, INT4_ASYM, GS 128, R 1.0, Scale | 16,845 | 0,039567696

Reply:
Smaller group size can also improve accuracy.